doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -883,14 +781,129 @@ the conversion in hypotheses :n:`{+ @ident}`.
       proof term. See `#12200
       <https://github.com/coq/coq/issues/12200>`_ for more details.
 
-Conversion tactics applied to hypotheses
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. _controlling-the-reduction-strategies:
+
+Controlling the reduction strategies and the conversion algorithm
+----------------------------------------------------------------------
+
+Coq provides reduction strategies that the tactics can invoke and two
+different algorithms to check the convertibility of types. The first
```

Comment:
```suggestion
Coq provides reduction strategies that the tactics can invoke and three
different algorithms to check the convertibility of types. The first
```
There's both `vm_compute` and `native_compute`.  Also make sure to update the following text.